ledger/error/src/test/suite/scala/com/daml/error/utils/ErrorDetailsSpec.scala

```diff
@@ -4,7 +4,7 @@
 package com.daml.error.utils
 
 import com.daml.error.ErrorCategory.BackgroundProcessDegradationWarning
-import com.daml.error.definitions.LedgerApiErrors
+import com.daml.error.definitions.CommonErrors
```

Comment:
Changed error examples used in this test in order to not depend on `//ledger/ledger-api-errors`